3.7/ubuntu/build

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+(time docker build -t rabbitmq:3.7 .) 2>&1 | tee "build.$(date -u +'%Y-%m-%d.%H%M%S').log"
```

Comment:
Not sure I understand why this script is checked in? (Same with the log file -- wouldn't it be better to add this variant to the Travis matrix so that Travis can give us build logs on every PR/branch/update?)